src/webextension/list/manage/components/app.css

```diff
@@ -50,6 +50,7 @@ body > main,
 
 .filter {
   border: 1px solid #d1d1d2;
+  border-bottom: 0;
```

Comment:
Let's remove this too; it should make everything look normal with the other removals above, and when we redo the search box, we can mess with this stuff as needed.